01_Data/src/layers/sequelize/repository/ChargingProfile.ts

```diff
@@ -37,7 +37,7 @@ export class SequelizeChargingProfileRepository extends SequelizeRepository<Char
     this.transaction = transaction ? transaction : new SequelizeRepository<Transaction>(config, Transaction.MODEL_NAME, logger, sequelizeInstance);
   }
 
-  async createOrUpdateChargingProfile(chargingProfile: ChargingProfileType, stationId: string, evseId: number, chargingLimitSource: ChargingLimitSourceEnumType, isActive?: boolean): Promise<ChargingProfile> {
+  async createOrUpdateChargingProfile(chargingProfile: ChargingProfileType, stationId: string, evseId?: number, chargingLimitSource?: ChargingLimitSourceEnumType, isActive?: boolean): Promise<ChargingProfile> {
```

Comment:
Thanks for the review, Christian!

> Could having the evseId optional cause issues if there are 2 EVSEs that have charging profiles? Or is that not a valid case? I see there is the unique constraint, which would technically avoid this.

According to the OCPP protocol, there should not be. 
The definition of charging profile id is simply "Required. Id of ChargingProfile." It should be unique in the scope of the charging station. Additionally, in K09.FR.03, it also mentions "a chargingProfileId uniquely identifies a charging profile". So id + stationId works as a unique constraint. 
Regarding 2 evses have the same charging profile, based on `evseId` field in `ReportChargingProfilesRequest`, it is a integer rather than a list. So I think this case will not happen. 🤔